src/mavlink-router/mainloop.cpp

```diff
@@ -30,6 +30,9 @@
 
 #include "autolog.h"
 
+#define TIMEOUT_LOG_SHUTDOWN_US     5e6 // number of microseconds we wait until we give up trying to stop log streaming
```

Comment:
5e6 is a double, can you use 5000000ULL?

Reply:
Done